docs/en/ingest-management/commands.asciidoc

```diff
@@ -25,6 +26,33 @@ run these commands.
 <hr>
 ++++
 
+[discrete]
+[[elastic-agent-status-command]]
+== elastic-agent status
+
+Returns the current status of the running Elastic Agent daemon.
+
+[discrete]
+=== Synopsis
+
+[source,shell]
+----
+elastic-agent status [--output <string>]
+                     [--help]
+                     [global-flags]
+----
+
+[discrete]
+=== Options
+
+`--output <string>`::
+Output the status information in either human, json, or yaml. (default "human")
```

Comment:
Would be nice to know what we mean by status information. We don't need an exhaustive list, just something to give the user some idea of what this command returns.